docs/en/mutations.md

```diff
@@ -52,17 +52,17 @@ You can also dispatch mutations using objects:
 
 ``` js
 store.dispatch({
-  type: 'INCREMENT',
-  payload: 10
+  type: 'INCREMENT_BY',
+  payload: [ 10 ]
```

Comment:
@stealthybox `console.log(...[[10]])` would be what you should be checking. I did not say that array of 1 element has special handling, but rather that the payload field contents IS the only element of the payloads array later on.